include/acutest.h

```diff
@@ -285,6 +285,12 @@
     #endif
 #endif
 
+/* Enable the use of the non-standard keyword CC_ATTRIBUTE to silence warnings under some compilers */
+#if defined(__GUNC__) || defined(__clang__) || defined(USE_CC_ATTRIBUTE)
+    #define CC_ATTRIBUTE(_x) __attribute__ ((_x))
```

Comment:
1. Could you please rename the macro e.g. to `TEST_ATTRIBUTE__` to follow the convention all the other private identifiers follow?

2. `__GUNC__` seems to be a typo.

Reply:
OK, fixed.  Apologies, that was indeed a typo.